server/cmd/dcrdex/log.go

```diff
@@ -50,16 +51,17 @@ var (
 	// It should be closed on application shutdown.
 	logRotator *rotator.Rotator
 
-	log           = backendLog.Logger("MAIN")
-	dbLogger      = backendLog.Logger("DB")
-	dexmanLogger  = backendLog.Logger("DEX")
-	commsLogger   = backendLog.Logger("COMM")
-	authLogger    = backendLog.Logger("AUTH")
-	swapLogger    = backendLog.Logger("SWAP")
-	marketLogger  = backendLog.Logger("MKT")
-	bookLogger    = backendLog.Logger("BOOK")
-	matcherLogger = backendLog.Logger("MTCH")
-	waiterLogger  = backendLog.Logger("CHWT")
+	log            = backendLog.Logger("MAIN")
+	dbLogger       = backendLog.Logger("DB")
+	dexmanLogger   = backendLog.Logger("DEX")
+	commsLogger    = backendLog.Logger("COMM")
+	authLogger     = backendLog.Logger("AUTH")
+	swapLogger     = backendLog.Logger("SWAP")
+	marketLogger   = backendLog.Logger("MKT")
+	bookLogger     = backendLog.Logger("BOOK")
+	matcherLogger  = backendLog.Logger("MTCH")
+	waiterLogger   = backendLog.Logger("CHWT")
+	webAdminLogger = backendLog.Logger("WADMN")
```

Comment:
Just `"ADM"` is find for the administration subsystem.  It just so happens to use http now, but at the core it's an administration interface and the protocol could be something else in the future.